json/schemav1.0/schema/schema-zfs-groupspace.json

```diff
@@ -0,0 +1,50 @@
+{
+    "$schema": "https://github.com/Alyseo/zfs/tree/json/json/schema/schema_zfs_groupspace.json",
+    "type":"object",
+    "name": "zfs groupspace -J",
+    "version": "1.0" ,
+    "description": "list all volume of filesystem",
+    "required":true,
+    "properties":{
+        "stderr": {
+            "type":"string",
+            "title": "stderr",
+            "description": "error output of command",
+            "required":true
+        },
+        "stdout": {
+            "type":"array",
+            "description": "standard output of command",
+            "minitems": "0",
+            "required":true,
+            "items":
+                {
+                    "type":"object",
+                    "type": "type of volume",
```

Comment:
`type` seems to be a duplicate key here, and overrides the schema restriction. Should "type of volume" be a property instead?